style.css

```diff
@@ -338,3 +356,111 @@ textarea {
 .get-in-touch:active {
   background-color: #f3f3f3;
 }
+
+.d-none {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] The value for min-width property should be in range of 992px - 1100px. These is the accepted desktop screen breakpoints. Change your value.